serving/ingress/pkg/reconciler/ingress/resources/route.go

```diff
@@ -132,6 +135,14 @@ func makeRoute(ci *networkingv1alpha1.Ingress, host string, rule networkingv1alp
 			WildcardPolicy: routev1.WildcardPolicyNone,
 		},
 	}
+
+	// If the passthrough annotation is set, target the HTTPS port and configure passthrough.
+	if _, ok := annotations[EnablePassthroughRouteAnnotation]; ok {
+		route.Spec.Port.TargetPort = intstr.FromString(HTTPSPort)
+		route.Spec.TLS.Termination = routev1.TLSTerminationPassthrough
+		route.Spec.TLS.InsecureEdgeTerminationPolicy = routev1.InsecureEdgeTerminationPolicyNone
```

Comment:
Just in case, `InsecureEdgeTerminationPolicyNone` is better than `InsecureEdgeTerminationPolicyAllow` here?
`oc get ksvc` prints `http://` but actually it is not available.

Reply:
hmm, good point. Maybe Redirect is indeed the safer choice?

Reply:
Yes, I think so.